vllm/model_executor/layers/quantization/compressed_tensors/schemes/compressed_tensors_w8a8_int8.py

```diff
@@ -46,10 +51,43 @@ def process_weights_after_loading(self, layer: torch.nn.Module) -> None:
                                            requires_grad=False)
         # INPUT SCALE
         if self.is_static_input_scheme:
-            layer.input_scale = Parameter(layer.input_scale.max(),
-                                          requires_grad=False)
+            if self.input_symmetric:
+                layer.input_scale = Parameter(layer.input_scale.max(),
+                                              requires_grad=False)
+            else:
+                # Static asymmetric quantization has not been tested yet.
+                # Kernel and ops support exists and is tested, it's just the
+                # following integration code that is untested.
+                logger.warning(
+                    "Static asymmetric quantization currently untested")
```

Comment:
Let's beef up this warning message a bit:
```suggestion
                logger.warning(
                    "Support for models with static asymmetric quantized activations has been implemented but is untested.")
```

Reply:
Hmm, do we have a nice way to only log once? This is going to cause a ton of spam if somebody actually runs into this

Reply:
Try this:
```suggestion
                vllm.utils.print_warning_once(
	                    "Support for models with static asymmetric quantized activations has been implemented but is untested.")
```